### ISSUE TITLE
Compilation error with OpenGLExample on Mono 5.0/Linux

### ISSUE THREAD

**Issue by MarkuBu** [rthome/SharpNoise#11]
If I try to compile the OpenGLExample I get this error message

```
/usr/lib/mono/xbuild/14.0/bin/Microsoft.Common.targets: Error: Tool exited with code: 1. Output: Error: Invalid ResX input.
Position: Line 123, Column 5.
Inner exception: Could not find a part of the path "/home/markus/Projects/SharpNoise/Examples/OpenGLExample/resources/normals.frag".
 (OpenGLExample)
```

Don't know what this means or how I can solve this.

**Comment by rthome** [comment]
Hi,

thanks for reporting this.

Some of the shader files for the OpenGL example, which are embedded in the app as ResX resources, were referenced with a lowercase path, `resources\shader.frag` instead of `Resources\shader.frag`. On Windows this is not an issue since paths are case-insensitive, but on Linux the casing matters.

I am also, conveniently, using Linux right now, so I was able to reproduce the error.

Should be fixed with commit 0a1ff844e70181610490e6b15863d89dadf5f1a9.

**Comment by MarkuBu** [comment]
Thanks for the quick response. Works fine now
